apps/right-to-rent-check/acceptance/features/landlord-agent.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('landlord agent step')
+
+Before((
+  I,
+  landlordAgentPage
+) => {
+  I.visitPage(landlordAgentPage, steps)
+});
+
+Scenario('I select Landlord, click ‘Continue’ & go to /landlord-details step', (
+  I,
+  landlordAgentPage,
+  landlordDetailsPage
+) => {
+  landlordAgentPage.selectLandlordAndSubmit();
```

Comment:
This totally makes sense and now I've amended. 

Now the question is do I even need the pages file for the variables? Again it requires me to open a separate file to understand some pretty simple variables